Assignment/Homework_1 - HTML/area.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Safei's Assignment 1</title>
+		<link rel="stylesheet" type="text/css" href="style.css">
+	</head>
+	
+	<header>
+			<button onclick="location.href='#dropdown'" style="margin-left:24px;">Dropdown</button>
+			<h1>Pokedex</h1>
+			<img src="assets/icons/pokeball_icon.png" style="float: right;margin-right:24px;margin-top: 22px;">
+	</header>
+	<body>
```

Comment:
everything being rendered should be in the body tag aka header should be within body